packages/create-llama/e2e/basic.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from "@playwright/test";
+import { exec } from "child_process";
+import { execSync } from "node:child_process";
+import crypto from "node:crypto";
+import { mkdir } from "node:fs/promises";
+import { fileURLToPath } from "node:url";
+import waitPort from "wait-port";
+import type {
+  TemplateEngine,
+  TemplateFramework,
+  TemplateType,
+  TemplateUI,
+} from "../templates";
+
+let cwd: string;
+test.beforeEach(async () => {
+  cwd = fileURLToPath(
+    new URL(`.cache/${crypto.randomUUID()}`, import.meta.url),
+  );
+  await mkdir(cwd, { recursive: true });
+});
+
+test("try create-llama", async ({ page }) => {
+  const createLlama = fileURLToPath(
+    new URL("../dist/index.js", import.meta.url),
+  );
+
+  const name = "basic";
+  const template: TemplateType = "streaming";
+  const framework: TemplateFramework = "nextjs";
+  const engine: TemplateEngine = "simple";
+  const ui: TemplateUI = "shadcn";
```

Comment:
@himself65 so currently we're testing only this configuration right? How about adding other permutations?

Reply:
I will add them later